### PR TITLE
THORN-1691: Introduce microprofile opentracing

### PR DESCRIPTION
**Pull request by pavolloffay**
https://issues.jboss.org/browse/THORN-1691

Microprofile OpenTracing fraction.

### CODE REVIEW

**Comment by pavolloffay** [comment]
@kenfinnigan @mkouba could you please review. It depends on final versions and all TCKs are passing locally.

File: fractions/microprofile/microprofile-opentracing/pom.xml [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    <groupId>org.wildfly.swarm</groupId>
+    <artifactId>build-parent</artifactId>
+    <version>2018.6.0-SNAPSHOT</version>
+    <relativePath>../../../build-parent/pom.xml</relativePath>
+  </parent>
+
+  <artifactId>microprofile-opentracing</artifactId>
+  <packaging>jar</packaging>
+
+  <name>Microprofile-OpenTracing</name>
```
Comment (Ladicek) [inline fractions/microprofile/microprofile-opentracing/pom.xml]: It's MicroProfile. All the MP fractions now have a consistent `<name>`, would be good to follow suit :-)

Comment (pavolloffay) [reply fractions/microprofile/microprofile-opentracing/pom.xml]: :+1: done

**Comment by kenfinnigan** [comment]
retest this please

**Comment by kenfinnigan** [comment]
retest this please

**Comment by pavolloffay** [comment]
@kenfinnigan @mkouba CI passed can it be merged?

**Comment by mkouba** [comment]
No please. I'd like to have https://github.com/thorntail/thorntail/pull/930 merged first. You will have to rebase and update the groupIds. That's the reason why I advised you to wait with the PR a few days until the master branch is stable ;-)

**Comment by pavolloffay** [comment]
Although this has a higher PR number, it's because it has been closed. 

I would like to get this merged ASAP. It's open insanely long time.

File: testsuite/microprofile-tcks/opentracing/pom.xml [hunk 2]
```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.thorntail.testsuite</groupId>
+    <artifactId>wildlfy-swarm-microprofile-tck-parent</artifactId>
```
Comment (mkouba) [inline testsuite/microprofile-tcks/opentracing/pom.xml]: This should be `thorntail-microprofile-tck-parent`.

File: testsuite/microprofile-tcks/opentracing/pom.xml [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    <version>2.0.0.Final-SNAPSHOT</version>
+    <relativePath>../</relativePath>
+  </parent>
+
+  <artifactId>wildlfy-swarm-microprofile-tck-opentracing</artifactId>
```
Comment (mkouba) [inline testsuite/microprofile-tcks/opentracing/pom.xml]: This should be `thorntail-microprofile-tck-opentracing`.

**Review by kenfinnigan — APPROVED** [review]

**Pull request merged**